functions_framework_builder/lib/src/utils.dart

```diff
@@ -0,0 +1,98 @@
+// Copyright 2021 Google LLC
```

Comment:
nit: 2020

See: 
https://github.com/google/new-project#source-code-headers

Reply:
All of our other ones are `2021` – not sure why @subfuzion went this way...

Reply:
Get ahead of the curve for next year?